utilities/tools/Set-ModuleReadMe.ps1

```diff
@@ -210,7 +210,7 @@ function Set-ParametersSection {
             # Add external single quotes to all default values of type string except for those using functions
             $defaultValue = ($parameter.defaultValue -is [array]) ? ('[{0}]' -f ($parameter.defaultValue -join ', ')) : (($parameter.defaultValue -is [hashtable]) ? '{object}' : (($parameter.defaultValue -is [string]) -and ($parameter.defaultValue -notmatch '\[\w+\(.*\).*\]') ? '''' + $parameter.defaultValue + '''' : $parameter.defaultValue))
             $allowedValue = ($parameter.allowedValues -is [array]) ? ('[{0}]' -f ($parameter.allowedValues -join ', ')) : (($parameter.allowedValues -is [hashtable]) ? '{object}' : $parameter.allowedValues)
-            $description = $parameter.metadata.description
+            $description = $parameter.metadata.description.Replace("`r`n", '<p>').Replace("`n", '<p>')
```

Comment:
QQ has this been used to regenerate all module readmes? Just to double check for any breaking changes although I don't think we are using multiline descriptions anywhere else. 

Besides that looks great :)

Reply:
The expectation is that no files are updated

Reply:
done

Reply:
A few files have been updated, but I don't think that was due to the readme generator change, was it? So it's probably due to module changes not including the readme update. How can we avoid that besides checking it while reviewing? Open question not strictly related to this PR

Reply:
No, its a general change. Presumably for modules that were updated before the new script was live / people did not use it

Reply:
Wouldn't know how else to check it unless we want to write more in-depth readme tests (e.g. do all columns have `''` in it if there is an empty default value, etc.)

Reply:
or maybe just run the readme generator in the validation step and fail if changes are added ?

Reply:
Feel free to create an issue for it. I guess we'd need to impement some sort of hash-comparison to detect if files were changed